desktop/api/src/main/java/org/datacleaner/actions/PreviewTransformedDataActionListener.java

```diff
@@ -196,8 +289,8 @@ public TableModel call() throws Exception {
     }
 
     private AnalysisJobBuilder copy(final AnalysisJobBuilder original) {
-        final AnalysisJob analysisJob = original.withoutListeners().toAnalysisJob(false);
-        final AnalysisJobBuilder ajb = new AnalysisJobBuilder(original.getConfiguration(), analysisJob);
-        return ajb;
+        final AnalysisJob analysisJob = original.getRootJobBuilder().withoutListeners().toAnalysisJob(false);
+        final AnalysisJobBuilder copy = new AnalysisJobBuilder(original.getConfiguration(), analysisJob);
```

Comment:
Not new code, but pretty funky going from builder to job to builder to make a copy :)